### PR TITLE
XeXe collision era and relval workflow (92X)

### PR DESCRIPTION
**Pull request by mandrenguyen**
Backport of #20749 20749
Customizes the pp reconstruction to keep timing in check.
To be used for XeXe data taking on October 12th.
Adds a relVal worklow to test (148)
Supersedes #20714

### CODE REVIEW

**Comment by mandrenguyen** [comment]
Can someone please trigger the tests?

**Comment by makortel** [comment]
@cmsbuild, please test

File: RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py [hunk 1]
```diff
@@ -44,5 +44,13 @@
 
 from Configuration.Eras.Modifier_peripheralPbPb_cff import peripheralPbPb
 peripheralPbPb.toModify(seedGeneratorFromRegionHitsEDProducer,
+                        doClusterCheck = True,  # FIXMETOO
```
Comment (perrotta) [inline RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py]: Shouldn't this doClusterCheck parameter remain inside the ClusterCheckPSet?

Comment (mandrenguyen) [reply RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py]: Indeed.  Didn't see it as that era isn't tested in relval.  As discussed with Slava we may merge or remove it soon.  Should I push a fix now, or are you still reviewing?

Comment (perrotta) [reply RecoTracker/TkSeedGenerator/python/SeedGeneratorFromRegionHitsEDProducer_cfi.py]: Please do it now,
I was comparing the 92X and 94X versions of your PR, and this is the only significant difference I spotted.

**Comment by davidlange6** [comment]
please test

**Comment by davidlange6** [comment]
merge

**Pull request merged**